Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1162,6 +1164,7 @@ def lhegensim(fragment,howMuch):
 steps['RECODR2_25nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_25ns']])
 steps['RECODR2_50nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_50ns']])
 steps['RECODR2_2016reHLT']=merge([{'--hltProcess':'reHLT','--conditions':'auto:run2_data_relval'},steps['RECODR2_2016']])
+steps['RECODR2newL1repack_2016reHLT']=merge([{'-s':'L1REPACK:FullSimTP,RAW2DIGI,L1Reco,RECO,EI,PAT,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign,DQM:@standardDQM+@miniAODDQM','--hltProcess':'reHLT'},steps['RECODR2_2016']])
```

Comment:
As this is a rebase of 80x PR, the above line does not use ```'--conditions':'auto:run2_data_relval'```
which in the meantime has been used in 7c6f44809f8bd39dec429a04ec588f967d3bee27 to line above.
@mmusich If you confirm, I can change the above line to use it, as: 
<pre>
+steps['RECODR2newL1repack_2016reHLT']=merge([{'-s':'L1REPACK:FullSimTP,RAW2DIGI,L1Reco,RECO,EI,PAT,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign,DQM:@standardDQM+@miniAODDQM','--hltProcess':'reHLT','--conditions':'auto:run2_data_relval'},steps['RECODR2_2016']])
</pre>
            
Mind you that ```'--conditions':'auto:run2_data_relval'``` is not used in the 80x PR #15117 [here](https://github.com/cms-sw/cmssw/pull/15117/files#diff-819af2ec677272fddf1fef50e788a88bR1142)
and if needed I can update it there also.  Let me know.